### ISSUE TITLE
SVG output option

### ISSUE THREAD

**Issue by strboul** [javierluraschi/nomnoml#3]
Thanks for porting *nomnoml* to R!
Are there any plans to have a SVG output option? For instance, the mermaid graphs from [DiagrammeR](https://cran.r-project.org/package=DiagrammeR) package supports SVG outputs, which is quite handy.

```r
DiagrammeR::mermaid("
graph LR
  A-->B
  A-->C
  C-->E
  B-->D
  C-->D
  D-->F
  E-->F
")
```

**Comment by andrie** [comment]
@javierluraschi, I also would love to have SVG output.  If you have the outline of an idea on how to achieve this, I can try to implement.

**Comment by andrie** [comment]
Aha, I can see it is possible to use the `svg` renderer in the dev version.

However, what I had in mind was the ability to get the SVG output file, so I can manipulate the SVG object downstream.

**Comment by andrie** [comment]
I think this was fixed in #12
